java/feature-layer-rendering-mode-scene/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        applicationId "com.esri.arcgisruntime.sample.featurelayerrenderingmodescene"
+        minSdkVersion 22
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
```

Comment:
`versionCode` and `versionName` should come from global vars

```groovy
versionCode rootProject.ext.versionCode
versionName rootProject.ext.versionName
```

This should be updated in the rendering mode map sample as well.